extensions/amp-app-banner/0.1/amp-app-banner.js

```diff
@@ -329,8 +330,9 @@ export class AmpAndroidAppBanner extends AbstractAppBanner {
     assertHttpsUrl(this.manifestHref_, this.element, 'manifest href');
 
     /** @private @const {!Element} */
-    this.openLink_ = user().assert(this.element.querySelector('a[open-link]'),
-        '<a open-link> is required inside %s: %s', TAG, this.element);
+    this.openButton_ = user().assert(
+        this.element.querySelector('button[open-link]'),
+        '<button open-link> is required inside %s: %s', TAG, this.element);
```

Comment:
maybe rename the attribute to `open-button` for consistency? (or `open-banner`)? It's a breaking change anyway, we might as well change the attribute too.